travis_build.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+set -ex
+
+COMMAND="./configure --strict && make && make test"
```

Comment:
don't you need to set the paths for cvmfs, python3, etc?

Reply:
My bad, forgot to add those. It looks like the current setup specifies

    --with-cvmfs-path /opt/libcvmfs --with-uuid-path /opt/uuid

correct?

Reply:
Yes, I think that should work.

For python3 we set it with configure.afs. I am not certain how to do extend that to this case, but perhaps python3 is automatically recognized in all of the containers?

Reply:
For the distros that use Python 3, it seems to be picking it up automatically. For the ones that use Python 2 (Centos 6 & 7), do we want to try to change it?

Reply:
For sure I know it picks python2 automatically for all of them, so let's try by only setting explicitly cvmfs and its dependencies (uuid).